anomaly/jsonxx.cc

```diff
@@ -0,0 +1,1163 @@
+// -*- mode: c++; c-basic-offset: 4; -*-
```

Comment:
How about putting this files to `thirdparty/jsonxx/jsonxx.cpp` (and the same for headers)? So we can share it with other subprojects, and we clearly separate our code from third party code